packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -2381,19 +2381,22 @@ void main() {
     await gesture.moveBy(const Offset(600, 0));
     // To the edge of the screen basically.
     await tester.pump();
+    await tester.pumpAndSettle();
```

Comment:
where do we introduce frame delays?

Reply:
There's no need for these changes. Removed.